src/org/ensembl/healthcheck/testcase/compara/CheckMethodLinkSpeciesSetTable.java

```diff
@@ -71,7 +71,7 @@ public boolean run(DatabaseRegistryEntry dbre) {
 			" method_link_species_set_id "+
 			" FROM method_link_species_set JOIN species_set USING (species_set_id)"+
 			" JOIN species_set_header USING (species_set_id)"+
-			" JOIN genome_db USING (genome_db_id) GROUP BY method_link_species_set_id";
+			" JOIN genome_db USING (genome_db_id) WHERE method_link_species_set.last_release IS NULL GROUP BY method_link_species_set_id";
```

Comment:
If we add: `AND first_release IS NOT NULL` it will make sure we only have the current species.